### ISSUE TITLE
Building with browserify

### ISSUE THREAD

**Issue by McOmghall** [jcassee/angular-hypermedia#24]
I'm trying to build my app with `browserify`, using node `require`s to get the required dependencies. Your main file for node doesn't seem to deal with the fact that `mediaTypeParser` and `linkHeaderParser` have to be global and therefore they have to be somehow included in the distributed files. Is there any way to change that to fit my build pipeline?

I'm currently solving it with the following lines but my opinion is the `node` build should work with `browserify` as it is, but I have no idea how to integrate this with your `gulp` build.

```
require('./node_modules/mediatype-parser/dist/mediatype-parser-browser');/*global mediaTypeParser*/
require('./node_modules/linkheader-parser/dist/linkheader-parser-browser');/*global linkHeaderParser*/
```

**Comment by McOmghall** [comment]
Solved it, you just need to add a ["browser" field](https://github.com/substack/node-browserify#browser-field) in your `package.json`. Sent pool requests:
https://github.com/jcassee/linkheader-parser/pull/1
https://github.com/jcassee/mediatype-parser/pull/1

**Comment by jcassee** [comment]
Fixed in jcassee/linkheader-parser@79b478d97ca1a7b4bdc59e69185108ac6ee418ad and jcassee/mediatype-parser@c55ec23355b4ceb47db4c8f5fc5f6fbdbd7221e2.
